### PR TITLE
Use VAS.Core.Catalog for translations

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: VAS.Core/Common/Enums.cs [hunk 1]
```diff
@@ -460,6 +460,7 @@ public enum PlayerViewOperationMode
 		LiveAnalysisReview,
 		Analysis,
 		SimpleWithControls,
+		Presentation,
```
Comment (vguzmanp) [inline VAS.Core/Common/Enums.cs]: Unrelated to the catalog

File: VAS.Core/Common/ExtensionMethods.cs [hunk 2]
```diff
@@ -235,5 +236,30 @@ public static IEnumerable<T> ToEnumerable<T> (this T item)
...
 			yield return item;
 		}
+
+		/// <summary>
+		/// Compare 2 dynamic objects.
+		/// </summary>
+		/// <returns><c>true</c> if both objects are equal</returns>
+		/// <param name="obj1">The first object to compare.</param>
+		/// <param name="obj2">The second object to compare.</param>
+		public static bool Compare (this ExpandoObject obj1, ExpandoObject obj2)
```
Comment (vguzmanp) [inline VAS.Core/Common/ExtensionMethods.cs]: Unrelated

Comment (ylatuya) [reply VAS.Core/Common/ExtensionMethods.cs]: I have pushed some commits from a different branch in this one
